libraries/AP_Filesystem/AP_Filesystem_Sys.cpp

```diff
@@ -31,7 +31,11 @@ struct SysFileList {
 };
 
 static const SysFileList sysfs_file_list[] = {
+#ifdef CH_DBG_STATISTICS
```

Comment:
#if HAL_ENABLE_THREAD_STATISTICS
#endif

and default that define based on CH_DBG_STATISTICS TRUE

Reply:
it would be nice to have a way to reset it

Reply:
Fixed